### DISCUSSION TITLE
❗ mark

### DISCUSSION THREAD

**Discussion by mgndnr1** [d4rken-org/sdmaid-se#974] [Q&A] [answered]
![IMG_20240128_114923.jpg](https://github.com/d4rken-org/sdmaid-se/assets/157942186/1d405ea5-724f-40b3-af96-e0984a7ce097)

What does this exclamation mark mean on my SD maid SE Foss 0.19.1

**Comment by d4rken** [comment, marked as answer]
Should be "Search and Delete in Onepass" from general options.
